pkgs/by-name/vo/vors/package.nix

```diff
@@ -0,0 +1,108 @@
+{
+  curl,
+  fetchurl,
+  lib,
+  genericUpdater,
+  go,
+  perl,
+  stdenv,
+  writeShellScript,
+  zstd,
+  pkg-config,
+  opusfile,
+  sox,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vors";
+  version = "3.1.0";
+
+  src = fetchurl {
+    url = "http://www.vors.stargrave.org/download/vors-${finalAttrs.version}.tar.zst";
+    hash = "sha256-ZRQI96j0n00eh1qxO8NgJeOQPU9bfzHoHa45xQNuzv8=";
+  };
+
+  buildInputs = [
+    go
+    opusfile
+    sox
+  ];
+
+  nativeBuildInputs = [
+    zstd
+    pkg-config
+    perl
+    makeWrapper
+  ];
```

Comment:
It would be ideal if this would build with `strictDeps = true` but there are issues finding `go`.